packages/fxa-content-server/app/scripts/templates/pair/index.mustache

```diff
@@ -6,7 +6,7 @@
   <section>
     <div class="error"></div>
 
-    <div class="graphic {{ graphicId }}"></div>
+    <div class="graphic {{ graphicId }}" role="img" aria-label="Connect another device"></div>
```

Comment:
`{{#t}}Scan to connect another device{{/t}}` (also, note the addition of the translation tag)